style.css

```diff
@@ -1,80 +1,284 @@
+/* COLORS:
+yellow: rgb(255, 240, 86);
+dark-black: rgb(25, 25, 25);
+dark-grey: rgb(157,157,157);
+light-grey: rgb(223, 226, 219);
+white: rgb(255, 255, 255);
+*/
+
 body {
-  background: url('./lokole-shot.jpg') no-repeat center center fixed;
-  background-size: cover;
-  background-color: #6e655c;
-  font-size: 24px;
-  line-height: 34px;
-  font-family: serif;
-  margin: 0;
+  overflow-y: hidden;
+  font-size: 16px;
+  text-align: center;
+  color: rgb(25, 25, 25);
+  background-color: rgb(223, 226, 219);
 }
 
-.page-container {
+#website-content {
+  display: none;
+}
+
+#modal-1-content,
+#modal-2-content {
+  display: none;
+}
+
+/* LAYOUTS */
+
+.row-layout {
   display: -webkit-box;
   display: -moz-box;
   display: -ms-flexbox;
   display: -webkit-flex;
   display: flex;
-  -webkit-box-pack: center;
-  -ms-flex-pack: center;
   justify-content: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
   align-items: center;
-  width: 100%;
-  height: 100vh;
+  align-content: center;
 }
 
-.content-box {
+.row-wrap-layout {
   display: -webkit-box;
   display: -moz-box;
   display: -ms-flexbox;
   display: -webkit-flex;
   display: flex;
-  -webkit-box-pack: center;
-  -ms-flex-pack: center;
+  flex-wrap: wrap;
   justify-content: center;
-  -ms-flex-line-pack: center;
+  align-items: center;
   align-content: center;
-  -webkit-box-orient: vertical;
-  -webkit-box-direction: normal;
-  -ms-flex-direction: column;
+}
+
+.vertical-layout {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
   flex-direction: column;
-  background-color: #fefefe;
-  padding: 20px 10px 20px 10px;
-  border-radius: 45px;
-  max-width: 30%;
-  text-align: center;
+  justify-content: flex-start;
+  align-items: start;
```

Comment:
Yes. But apparently it works even if you just use "start". See [this tutorial](https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_align-items).

I'll change it to flex-start to avoid problems though.